cmd/util/printer/print.go

```diff
@@ -413,8 +413,7 @@ func summariseHistoryEvents(history []*models.JobHistory) []models.Event {
 	for _, entry := range history {
 		hasDetails := entry.Event.Details != nil
 		failsExecution := hasDetails && entry.Event.Details[models.DetailsKeyFailsExecution] == "true"
-		terminalState := entry.ExecutionState.New.IsTermainl()
-		if (failsExecution || terminalState) && entry.Event.Message != "" {
+		if failsExecution && entry.Event.Message != "" {
```

Comment:
Is cancellation of execution treated as "Failed" ? What I mean to say is that now since we have removed `isTerminal` check, cancellation of execution would not be part of the summary.

Reply:
We are still logging a meaningful message when the job is cancelled. Not sure why we were checking for execution's terminal state

```
→ bacalhau docker run ubuntu sleep 120
Job successfully submitted. Job ID: j-b1a7a4df-9d4d-4785-821e-03780d532202
Checking job status... (Enter Ctrl+C to exit at any time, your job will continue running):

	Communicating with the network  ................  done ✅  0.0s
	   Creating job for submission  ................  done ✅  0.5s
	               Job in progress  ................  err  ❌  6.5s

Error: Stopped at user request


To get more details about the run, execute:
	bacalhau job describe j-b1a7a4df-9d4d-4785-821e-03780d532202

To get more details about the run executions, execute:
	bacalhau job executions j-b1a7a4df-9d4d-4785-821e-03780d532202
```